dev-client/__tests__/snapshot/__snapshots__/SiteTabsScreen-test.tsx.snap

```diff
@@ -1987,7 +1987,8 @@ exports[`renders correctly 1`] = `
                                       }
                                     >
                                       Top Match
-                                      : 
+                                      :
+                                       
```

Comment:
It's because of my refactoring to the code that uses `SoilIdStatusDisplay`; you can take a look at it if you want, basically something moved from outside a nested <Text> tag to inside. Unless it results in undesirable visual artifacts (I didn't notice any?) I don't think it matters.